ios/Capacitor/Capacitor/WebViewAssetHandler.swift

```diff
@@ -6,6 +6,7 @@ import MobileCoreServices
 open class WebViewAssetHandler: NSObject, WKURLSchemeHandler {
     private var router: Router
     private var serverUrl: URL?
+    private var pendingTasks: [Int] = []
```

Comment:
I would suggest making this a Set instead of an Array. Set.contains is an O(1) operation versus Array being an O(n).